controllers/homeRoutes.js

```diff
@@ -53,7 +53,25 @@ router.get('/space/:id', async (req, res) => {
 router.get('/space/:space_id/idea', withAuth);
 
 // View a specific idea
-router.get('/idea/:idea_id', withAuth);
+router.get('/idea/:id', withAuth, async (req, res) => {
+  try {
+    const ideaData = await Idea.findByPk(req.params.id);
+    const commentData = await Comment.findAll({
+      where: {
+        idea_id: ideaData.id
+      }
+    });
+
```

Comment:
The comment data here should be loaded into the Idea through an include rather than a separate query. See the Space + Ideas example for the space route.